.github/workflows/lint.yml

```diff
@@ -0,0 +1,34 @@
+---
+name: Run Linters
+on:
+  - push
+jobs:
+  linters:
+    name: Run Linters
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: "3.8"
+
+      - name: Run ansible-lint
+        uses: ansible/ansible-lint-action@master
+        with:
+          targets: |
+            tests/*.yml
+            tests/*/*.yml
+            tests/*/*/*.yml
+            playbooks/*.yml
+            playbooks/*/*.yml
+        env:
+          ANSIBLE_MODULE_UTILS: plugins/module_utils
+          ANSIBLE_LIBRARY: plugins/modules
+
+      - name: Run yaml-lint
+        uses: ibiqlik/action-yamllint@v1
+
+      - name: Run flake8
+        uses: julianwachholz/flake8-action@v1
+        with:
+          checkName: "Python Lint"
```

Comment:
I'd use "flake8" in the name. There is a bunch of different python lints available. In this case I'd say it's better to be more specific.

Reply:
The whole Python Lint process have changed.